### DISCUSSION TITLE
Integración con proveedor de firma electrónica (Docusign)

### DISCUSSION THREAD

**Discussion by NestorFGG** [SINCOSOFT-SAS/SGD#46] [General]
La integración con un nuevo proveedor de firma electrónica de documentos, permitiría ampliar el alcance de integración para lograr ubicar dentro de SincoSGD los documentos firmados electrónicamente desde un aplicativo externo

**Comment by NestorFGG** [comment]
Se realiza reunión con el tercero con el fin de aclarar los cambios en sus API para habilitar la integración, por tanto, la actividad en este momento está a cargo de ellos y estaremos a la espera de la entrega de los API o novedades respecto a lo solicitado por SINCO SGD. Todo se canalizará a través de correo electrónico, ya que este desarrollo nace de un REQ de nuestro cliente AR Construcciones.

**Comment by aleja550** [comment]
Nunca se realizo ni tampoco está en planeación del producto actual. La integración es administrada por CBR
